### DISCUSSION TITLE
Default credentials - user creation

### DISCUSSION THREAD

**Discussion by snakka-hv** [JordanKnott/taskcafe#126] [General]
I am trying docker-compose -p taskcafe up -d as per docker-compose instructions (latest image). Am able to access the application but am unable to figure if there is a default credential set to use for the initial login.
* Unable to register/create a user from login screen.
* Did not find any references to credentials in docker compose files
* Did not find any references in wiki or readme related to default/admin user.

Any help or pointers?

**Comment by JordanKnott** [comment]
It should automatically take you to a user registration screen - alternatively you can go directly to `<url>/register`

**Reply by saikrishna1488** [reply]
register button is not working

**Reply by mihaiserban92** [reply]
the register button is still not working for new deployments
